### PR TITLE
jumbomoji size calculation code

### PR DESCRIPTION
**Pull request by Simon-Laux**
somehow I can not get it to change the actual font size though :(

### CODE REVIEW

File: deltachat-ios/Chat/Views/Cells/TextMessageCell.swift [hunk 1]
```diff
@@ -28,3 +50,27 @@ class TextMessageCell: BaseMessageCell {
...
+    }
+    var isCombinedIntoEmoji: Bool {
+        unicodeScalars.count > 1 && unicodeScalars.first?.properties.isEmoji ?? false
+    }
+    var isEmoji: Bool { isSimpleEmoji || isCombinedIntoEmoji }
+}
+
+extension String {
+    var containsOnlyEmoji: Bool {
+        return !isEmpty && !contains { !$0.isEmoji }
```
Comment (cyBerta) [inline deltachat-ios/Chat/Views/Cells/TextMessageCell.swift:74]: Don't we want to restrict the scaling to 3 emojis?

File: deltachat-ios/Chat/Views/Cells/BaseMessageCell.swift [hunk 2]
```diff
@@ -452,7 +452,7 @@ public class BaseMessageCell: UITableViewCell {
             quoteView.isHidden = true
         }
 
-        messageLabel.attributedText = MessageUtils.getFormattedSearchResultMessage(messageText: msg.text,
+        messageLabel.attributedText = MessageUtils.getFormattedTextMessage(messageText: msg.text,
```
Comment (cyBerta) [inline deltachat-ios/Chat/Views/Cells/BaseMessageCell.swift:455]: attibuted text always overwrite simple texts: https://developer.apple.com/documentation/uikit/uitextview/1618626-attributedtext !

**Comment by r10s** [comment]
looking forward to see this in action! this is a really nice improvement!

**Review by cyBerta — APPROVED** [review]

**Comment by r10s** [comment]
there is a build error for me on xcode 14.2:

Screenshot 2023-02-11 at 15 53 52

is it building on your machines?

**Comment by cyBerta** [comment]
Weird, it did, going to double check that on monday.

**Comment by Simon-Laux** [comment]
I forgot to commit the last changes it seems 🤔.
I pushed them just now

**Comment by r10s** [comment]
cool stuff, thanks a lot!

**Pull request merged**